com.unity.multiplayer.mlapi/Tests/Runtime/MultiInstance/MultiInstanceHelpers.cs

```diff
@@ -62,6 +68,25 @@ public static bool Create(int clientCount, out NetworkManager server, out Networ
             return true;
         }
 
+
+        public static void ShutdownAndClean()
```

Comment:
minor: I'd name this as `Destroy` to keep it similar to `Create`/`Start` formats.

```suggestion
        public static void Destroy()
```

Reply:
I really don't have a preference to what we name this... 
@TwoTenPvP What do you want this to be named?